### ISSUE TITLE
Chrome Blocked Zip, Calls it a Virus

### ISSUE THREAD

**Issue by PatchedSamoyed** [CloneTrooper1019/Roblox_0.3.368.0#10]
Here is the link to the VirusTotal scan: https://www.virustotal.com/gui/file/5a0b687cbc73f44d16a8a77821cf13055e4b2c8e8fec9624bc2777047cedbeba/detection

**Comment by CloneTrooper1019** [comment]
Probably some random misdiagnosis since it isn’t being picked up by any other antivirus vendor. Don’t know what to tell you there.

**Comment by JustDoom** [comment]
same, but with microsoft edge too

**Comment by CloneTrooper1019** [comment]
Are you guys seeing this with the latest deployed version?
https://github.com/CloneTrooper1019/Roblox_0.3.368.0/archive/master.zip

**Comment by 39p** [comment]
> Are you guys seeing this with the latest deployed version?
> https://github.com/CloneTrooper1019/Roblox_0.3.368.0/archive/master.zip

Decided to check, it gives me the same result from the link you have the sent. Only malwarebytes detects it as malware. https://www.virustotal.com/gui/file/31010d1eff91af1f67806fa90fb19c056e3d0cb0531fea6dd40853390f43d458/detection

**Comment by MeDerpWasTaken** [comment]
Relating to this, windows defender claimed the zip file was a virus and blocked it

It detects it as `Trojan:Win32/Spursint.F!cl`

**Comment by JustDoom** [comment]
it is safe though right?

**Comment by bogotesr** [comment]
> it is safe though right?

yes, its weird you guys are having antivirus issues, I didn't have any issues.

**Comment by CloneTrooper1019** [comment]
The reason this is happening is because Roblox didn't sign the application in 2007. 
Pretty much every unsigned application that does anything today is marked as malicious even if it isn't doing anything shady.
It's super annoying but there's not much that can be done.
